ipinfo/cmd_tool_prefix.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+var completionsToolPrefix = &complete.Command{
+	Sub: map[string]*complete.Command{
+		"addr": completionsToolPrefixAddr,
+	},
+	Flags: map[string]complete.Predictor{
+		"-h":     predict.Nothing,
+		"--help": predict.Nothing,
+	},
+}
+
+func printHelpToolPrefix() {
+	fmt.Printf(
+		`Usage: %s tool prefix <cmd> [<opts>] [<args>]
+
+Commands:
+	addr	returns prefix's IP address.
```

Comment:
Note to not use real tab chars

Reply:
Also the help message can be a bit more specific, what IP address exactly?

Reply:
so, it just return the IP address of p (here p is Prefix Struct type) for example if the input is = 8.8.8.8/12 output will be 8.8.8.8